docs/src/conf.py

```diff
@@ -1,57 +1,80 @@
-# Configuration file for the Sphinx documentation builder.
-#
-# For the full list of built-in configuration values, see the documentation:
-# https://www.sphinx-doc.org/en/master/usage/configuration.html
-
 import os
-import re
+import sys
+from datetime import datetime
+
+import tomli  # Replace by tomllib from std library once docs are build with Python 3.11
+
+import metatensor_models
 
 
+ROOT = os.path.abspath(os.path.join("..", ".."))
+sys.path.insert(0, ROOT)
+
 # -- Project information -----------------------------------------------------
-# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
-project = 'metatensor-models'
-copyright = '2023, metatensor-models developers'
-author = 'metatensor-models developers'
+# The master toctree document.
+master_doc = "index"
 
-# Parse the version from the module.
-with open(os.path.join(os.path.dirname(__file__), '..', '..', 'src', 'metatensor_models', '__init__.py')) as f:
-    version = re.match(r".*__version__ = \"(.*?)\"", f.read(), re.S).group(1)
-release = version
+with open(os.path.join(ROOT, "pyproject.toml"), "rb") as fp:
+    project_dict = tomli.load(fp)["project"]
 
+project = project_dict["name"]
+author = ", ".join(a["name"] for a in project_dict["authors"])
 
-# -- General configuration ---------------------------------------------------
+copyright = f"{datetime.now().date().year}, {author}"
 
-needs_sphinx = "4.0.0"
+# The full version, including alpha/beta/rc tags
+release = metatensor_models.__version__
 
-python_use_unqualified_type_names = True
+
+# -- General configuration ---------------------------------------------------
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx_rtd_dark_mode',
```

Comment:
How did you manage to keep the dark mode even though you deleted this?

Reply:
I would keep the dark mode as the default. In terms of marketing, this is a very simple change that would allow us to stand out from all the packages which use the default themes at near-zero additional effort. I've also noticed that a lot of display issues (things that look weird in light mode) disappear in dark mode

Reply:
Darkmode is still the default. Note that we are not using the RTD docs theme anymore but [furo](https://github.com/pradyunsg/furo) (which we also use the other lab-cosmo packages).

Reply:
Ok, perfect!